packages/terra-application-navigation/src/common/_PopupMenu.jsx

```diff
@@ -193,6 +202,7 @@ const PopupMenu = ({
         <ul className={cx('utility-list')} aria-label={title} ref={listRef} role={role} tabIndex="0" onKeyDown={handleKeyDown}>
           {menuItems.map(item => (
             <PopupMenuListItem
+              id={item.id || undefined}
```

Comment:
```suggestion
              id={item.id}
```

Reply:
From what I understand the ``` || undefined ``` prevents id from being put in the dom if item.id is null. However, if there is a more standard way to do that I'll change it.